features/bevragen/persoon-beperkt/rni/dev/overzicht-gba.feature

```diff
@@ -144,7 +144,7 @@ Rule: RNI-deelnemer gegevens die horen bij categorie 01 (Persoon) en/of 08 (Verb
     | geslachtsnaam | Jansen |
     En heeft de persoon de volgende 'verblijfplaats' gegevens
     | naam              | waarde |
-    | land.code         | 5010   |
+    | land.c  ode         | 5010   |
```

Comment:
spaties in `c  ode` moeten weg

Reply:
hoe dat er nou ingekomen is??? is nu gecorrigeerd